apps/preview/nuxt/pages/examples/SfScrollable.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <ComponentExample :controls-attrs="controlsAttrs">
+    <SfScrollable
+      :drag="state.drag"
+      :direction="state.direction"
+      :buttons-placement="state.buttonsPlacement"
+      :class="[
+        'w-full items-center',
+        {
+          'snap-x snap-mandatory': state.snap,
+          'scrollbar-hide': state.hideScrollbar,
```

Comment:
What do you think about using `[&::-webkit-scrollbar]:hidden [-ms-overflow-style:'none'] [scrollbar-width:'none'] ` instead of  tailwind-scrollbar-hide lib? I tested it on Gallery and it works.

Reply:
your suggestion is probably better, @FRSgit added this library, but its extra dependency and from direct classes we already see whats happening https://github.com/vuestorefront/storefront-ui/pull/2571/commits/037a723863d67b3e271dcb64488c82f8a23017a4